### DISCUSSION TITLE
Self hosted Github Actions Runner on DELL ECS - No external Clouds restriction

### DISCUSSION THREAD

**Discussion by MalteBoehm** [community/community#18658] [Actions]
Hello is it possible to host a runner on a DELL ECS. We have the restriction that access to clouds like AWS or Google are basically prohibited. Also the storage of logs etc. should be possible in our own cloud. 

Looking forward to your feedback
Malte

**Comment by 5HT2** [comment]
Yes, it is possible.

**Reply by MalteBoehm** [reply]
I got the info from a team member that S3 or another cloud was once necessary and the way was not yet clear for us, so to speak. Thanks for your info!

**Reply by MalteBoehm** [reply]
> Yes, it is possible.


Are you sure? Documentations says:
GitHub Actions supports these storage providers:

Azure Blob storage
Amazon S3
S3-compatible MinIO Gateway for NAS

https://docs.github.com/en/enterprise-server@3.4/admin/github-actions/getting-started-with-github-actions-for-your-enterprise/getting-started-with-github-actions-for-github-enterprise-server#external-storage-requirements
